tests/scenarios.go

```diff
@@ -185,6 +186,12 @@ func GetResponse(ctx gobdd.Context) []reflect.Value {
 	return r.([]reflect.Value)
 }
 
+// GetResponseStatusCode returns request response status code.
+func GetResponseStatusCode(resp []reflect.Value) int {
+	// Execute() returns triples -> 2nd value is *http.Response -> get StatusCode
+	return resp[len(resp)-2].Interface().(*http.Response).StatusCode
```

Comment:
shall we try to return `(int, err)` in case `resp[len(resp)-2].Interface().(*http.Response) == nil`?